### ISSUE TITLE
Internationalization: lang: de-DE

### ISSUE THREAD

**Issue by mariusbarth** [crsh/papaja#550]
Hi @crsh,

I just tried to render the standard **papaja** template with `lang: de-DE`. Some but not all text is changed to German, see attached [pdf](https://github.com/crsh/papaja/files/10499515/test.pdf).

- What works:
  - *Abstract* is replaced with *Zusammenfassung*.
  -  Ampersands are replaced with *und*
- What doesn't work:
  - *Running Head*
  - *Author Note*
  - *Keywords*, *Word count*
  - ...

**Comment by crsh** [comment]
Yes, this is indeed a bit a pesky issue. We have some [legacy code](https://stackoverflow.com/questions/75906890/accessing-language-specific-fixed-names-in-papaja-terms) that doen't really do much anymore. I have just pushed some work in progress on the `langs` branch that should address keywords and word count. I think the rest needs to be fixed via LaTeX:

~~~latex
\renewcommand{\rheadname}{Kurztitel}% Running head
\renewcommand{\acksname}{Autorenhinweis}% Author Note
%\renewcommand{\keywordname}{Schlagw\"orter}% Keywords
\renewcommand{\notesname}{FuÃŸnoten}% Footnotes
~~~

The challenge is that I was only able to get this to work by manually adding this to the generated LaTeX-file in the document body before `\maketitle`. So I think this will require some Lua.

Another thing that is currently not localized is text generated from metadata:

- Contributorship line
- Correpsondence line
